main.go

```diff
@@ -20,10 +20,11 @@ var (
 	TOOL_NEO_GO      = "neo-go"
 	TOOL_NEO_EXPRESS = "neo-express"
 
-	LANG_GO     = "go"
-	LANG_PYTHON = "python"
-	LANG_JAVA   = "java"
-	LANG_CSHARP = "csharp"
+	LANG_GO         = "go"
+	LANG_PYTHON     = "python"
+	LANG_JAVA       = "java"
+	LANG_CSHARP     = "csharp"
+	LANG_TYPESCRIPT = "typescript"
```

Comment:
Just a thought, `typescript` is quite long to type and I believe it's often referred to as just "ts". We could label it "ts" so you get
```
cpm generate ts <options> <args>
```